src/commands/service/repl.ts

```diff
@@ -147,13 +147,12 @@ Call ${facadeModuleConfig.name} service functions in repl like this:
 
 ${color.yellow(
   `call ${facadeModuleConfig.name} <function_name> [<arg1>, <arg2>]`,
-)}${SEPARATOR}}`);
+)}${SEPARATOR}`);
 
     await haltCountly();
 
     spawn(mreplPath, [fluenceTmpConfigTomlPath], {
       stdio: "inherit",
-      detached: true,
```

Comment:
let's probably replace this with e.g. `!commandObj.config.windows`, cause on linux it works nice as it is

Reply:
I also tested that on linux and haven't noticed any difference. So i'm curious why did you initially made that change?

Reply:
I think the idea behind it was for the telemetry we send to Countly when command ends execution. but without detach command never ends execution. It's totally minor, but still

Reply:
seems like I am mistaken cause in the code I can see
`await haltCountly();` before this runs
maybe it doesn't matter then
Probably fine to remove this detached if it doesn't really affect anything anyway

maybe the main idea was to just end nodejs process cause it's not needed anymore, it probably hangs in the background without detached

Reply:
Reading the [doc ](https://nodejs.org/api/child_process.htm)right now and it says that:

- By default, the parent will wait for the detached child to exit. To prevent the parent from waiting for a given subprocess to exit, use the subprocess.unref() method.

- When using the detached option to start a long-running process, the process will not stay running in the background after the parent exits unless it is provided with a stdio configuration that is not connected to the parent. If the parent's stdio is inherited, the child will remain attached to the controlling terminal.

Seems like we need to do additional work/research to achieve desirable result. Mind if we do that in separate PR?

Reply:
sure. nice. I really don't remember why I did it. There was some reason definetly, cause it's a weird thing to add